filters/cookie/cookie_test.go

```diff
@@ -382,3 +383,101 @@ func TestSetCookie(t *testing.T) {
 		}
 	}
 }
+
+func TestDropCookie(t *testing.T) {
```

Comment:
If this filter could have only one arg, that's okay. If not, we need to test filter with two args as well.

Reply:
Yes right now it's only one arg allowed